mautic/mautic
# Multiscoring

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | mautic/documentation#214
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #581
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Multiscoring; allows users to create multiple score categories, and using them in actions / triggers.
A score category may (or may not) update the global score.
Therefore a company may use the multiscoring to gauge leads' interest on a particular solution and/or aspect of a solution.

#### Steps to test this PR:
1. Apply the PR
2. Create score categories (if not admin, you should check permissions before)
3. Use a score category in an action in a form (for example)
4. Test the form

A complete documentation with tests will be released and bound to that PR later.
